bedrock/firefox/templates/firefox/personal.html

```diff
@@ -37,6 +37,9 @@
     <a rel="external" href="{{ settings.GOOGLE_PLAY_FIREFOX_LINK }}">
       {{ high_res_img('firefox/android/btn-google-play.png', {'alt': 'Get it on Google Play', 'width': '172', 'l10n': True}) }}
     </a>
+    <small class="download-other">
+      <a href="{{ url('firefox.android.all') }}">{{_('Systems &amp; Languages')}}</a>
+    </small>
```

Comment:
So, the off white is fine, as long as we bump the font size as well. I have to say that the links under the normal download button personally, is to small as well and the line-height is much to tight but, this is up for an overhaul so, I will leave that as is for now :)

I would ideally here go for white, bump the size a little and, if there is concern that it will not seem like a link (although I believe we are safe), we can give it an underline by default. Thoughts?